Reddit - Remove "Advertise", "Get Premium" section

ZivSimchoni opened the pull request.

ZivSimchoni committed:
Reddit - Remove "Advertise", "Get Premium" section

mig4ng commented on reddit.txt:
```diff
@@ -0,0 +1,5 @@
+!! Reddit
+! Advertise button top bar
+www.reddit.com##._24UNt1hkbrZxLzs5vkvuDh
```
These ids are very likely to change and are very brittle. Use the human readable class names from elements near it and use `:upwards(N)` to go up the element nest. Or similar methods to get to the element we want to remove without using this likely to change.

Thanks for the contribution ❤️

ZivSimchoni replied:
I've updated it using the [aria-label], this seems to provide the same result.
Regarding the sidebar premium banner, I have yet to find any human-readable text that hides it.

Please review the new commit.

ZivSimchoni committed:
Update reddit.txt

This seems to provide the same result.
Regarding the sidebar premium banner, I have yet to find any human-readable text that hides it.
